README.md

```diff
@@ -1,212 +1,122 @@
 
-#### Installing mockery
-There are multiple options to install mockery. Use any one of the following to download.
-##### Download the mockery binary
-Use the release page link [mockery releases](https://github.com/vektra/mockery/releases/tag/v2.39.1) to download the artifact for your respective device.
-##### Using go install
-If you have go already installed on your device you can use the go install command to download mockery.
 ```sh
-go install github.com/vektra/mockery/v2@v2.39.1
+docker build --no-cache -t sphinx-tribes .
+docker tag sphinx-tribes sphinxlightning/sphinx-tribes:x
+docker push sphinxlightning/sphinx-tribes:x
 ```
 
-##### Using homebrew
-If you are on mac you can use homebrew to download mockery
-```zsh
-brew install mockery
-brew upgrade mockery
-```
+### Environment Configuration
 
-#### When adding a new function to the interface which is already mocked follow the below steps:
-1. Update the corresponding interface with the function signature, for example if you are adding new function to the ```database``` structure make sure the interface file ```db/interface.go``` is updated with the function signature.
-2. run the command ```mockery``` to update the mocks.
-
-
-#### To create mocks for a new interface make follow the steps below:
-
-1. Add the new entry in the ```.mockery.yml``` file like this
-```yml
-with-expecter: true
-dir: "mocks"
-packages:
-    github.com/stakwork/sphinx-tribes/db:
-        interfaces:
-            Database:
-    github.com/stakwork/sphinx-tribes/*your-package-name*:
-        interfaces:
-            *your-interface-1*:
-            *your-interface-2*:
-```
-2. run the command ```mockery``` to update the mocks.
+Create a `.env` file in the project root with the required environment variables.
 
+### Database Setup
 
-### Run Test
+Set up a PostgreSQL database and execute the provided SQL scripts to create necessary tables.
 
-```
-    // you may need to install cover with this command first
-    // go get golang.org/x/tools/cmd/cover
-    go test ./... -tags mock -race -v -coverprofile=coverage.out && ./cover-check.sh coverage.out <min coverage amount> 
-    // To get code coverage in html format do the following after running the code above
-    go tool cover -html="coverage.out"
-```
+### Running the Backend
 
-### Enable Relay
+Build and run the Golang backend:
 
-If you would like to enable Relay for invoice creation and keysend payment add the relay env key and values to the .env file
-
-```
-    RELAY_URL=
-    RELAY_AUTH_KEY=
+```sh
+go build .
+./sphinx-tribes
 ```
 
-### Enable Meme Image Upload
+## Optional Features
 
-If you would like to enable Meme image upload for organization's add the meme env key and value to the .env file,
-Meme image upload works with Relay enabled, so a running Relay is required for Meme to work.
+### Redis for Caching
 
-```
-    MEME_URL=
-```
+Configure Redis by adding the `REDIS_URL` or other relevant variables to your `.env` file.
```

Comment:
Can you specify the other relevant variables? 
```
REDIS_HOST=
REDIS_USER=
REDIS_PASS=
REDIS_DB=
```

To make it easier for developers.